Scrabble.py

```diff
@@ -36,21 +58,29 @@
                     a += line
                     # panaikina pasikartojančius žodžius
                     if temp != line:
-                        print(line)
+                        # Įdeda žodį ir žodžio vertę į lines_scored dictionary
+                        line_word = line
+                        score = get_value(line)
+                        line_scored[line_word] = score
                     temp = line
 # nuima dar vieną raidę, tai jau 3 nuimtos
                 for e in range(5):
                     removed3 = raides.pop(e)
                     if sorted(line) == raides:
                         a += line
-                        # panaikina pasikartojančius žodžius
+                        # panaikina pasikartojančius žodžius ir parašo kiek taškų vertas žodis.
                         if temp != line:
-                            print(line)
+                            # Adds found word to a line_scored dictionary
+                            line_word = line
+                            score = get_value(line)
+                            line_scored[line_word] = score
                         temp = line
                     raides.append(removed3)
                     raides = sorted(raides)
                 raides.append(removed2)
                 raides = sorted(raides)
             raides.append(removed)
             raides = sorted(raides)
+(print(*[str(k) + " " + str(v) + " taškai" for k, v in
+         sorted(line_scored.items(), key=lambda x: x[1], reverse=True)], sep='\n'))
```

Comment:
patikrinau, kad veikia, tačiau nieko nesuprantu, prašau parašyk logiką kaip komentarą ir koks šitų eilučių tikslas

Reply:
Pridėjau komentarą kuris paaiškina veikimą.